### PR TITLE
feature/parent-contact-page

### PR DESCRIPTION
**Pull request by MosharrafMusa**
# Pull Request Template

## Description
Add a summary that address the following:
 - motivation/purpose for the feature 
 Create a separate contact us page so that user can communicate with the authority
 - What were those changes 
 Match up with other page structure and design
 - What issues were fixed 
Before this page was build followed by old figma wirefame, now it is updated with current figma design.
 - Provide the link to the user story it is referencing in Trello board or other supporting document
contact us
## Commits Checklist
- [x] commits have clear title names 
- [x] commit has descriptive message of what has changed 
- [x] more small commits than less big commits 

## Code Checklist 
- [x] code is formatted appropriately 
- [x] code meets DRY standards
- [x] no dead code (check for logs and print statements)  
- [x] no TODOS instead necessary comments 
- [x] code follows Story Squad standards for:
     - Language 
     - Framework 
     - Libraries   

### PRs needs to be reviewed by at least 2 team members
 - TPL is the 3rd reviewer 
 - Lots of comments and suggestions, please! 

[Pull Request Rubric](https://www.notion.so/1fc04e4fedeb429ba873b7c68d281707?v=74054da7991341c0bf970f39410c43da)

### CODE REVIEW

File: src/components/pages/ParentContact/ParentContactContainer.js
Comment (Robert-Lark) [inline src/components/pages/ParentContact/ParentContactContainer.js:25]: Curious to know what is going on in this area of the code - it might be good to comment this section for incoming teams also.

File: src/components/pages/ParentContact/ParentContactContainer.js
Comment (Robert-Lark) [inline src/components/pages/ParentContact/ParentContactContainer.js:31]: Looks like you aren't following the file structure of the rest of the application - I would move all of this into a renderParentContact file and then import it into the ParentContact container. This way of settings things up is called the redux container pattern - you can learn about it here https://www.thegreatcodeadventure.com/the-react-plus-redux-container-pattern/

File: src/components/pages/ParentContact/ParentContactContainer.js
Comment (Robert-Lark) [inline src/components/pages/ParentContact/ParentContactContainer.js:69]: Looks like you are using ant-d for everything except the form itself? What was the reasoning behind this technical decision?

File: src/components/pages/ParentContact/ParentContactContainer.js
Comment (Robert-Lark) [inline src/components/pages/ParentContact/ParentContactContainer.js:122]: What was the reasoning behind keeping the image below inside the parentContact folder as opposed to in the public folder with the rest of the images?

Comment (Robert-Lark) [reply src/components/pages/ParentContact/ParentContactContainer.js:122]: if you do end up moving it over there you might want to give it a more descriptive name then 'img2'

File: src/index.js
Comment (Robert-Lark) [inline src/index.js:203]: I don't think you need the 'exact' keyword in here as that only applies to '/' URLs - I might be mistaken about that though so - if someone else can back me up here that would be great.

File: src/styles/less/ParentContact.less
Comment (Robert-Lark) [inline src/styles/less/ParentContact.less:8]: nice use of the variables here :)

**Review by Robert-Lark — COMMENTED** [review]
There are a couple of things in here that might need another set of eyes before it can get merged in - I have seen this on a couple of PR's now but you have -26,000 lines of code committed - looks like it has something to do with the package-lock.json.

**Review by leachtucker — APPROVED** [review]
Looks great! Nice work of utilizing the mixin styles here.

**Review by Patrick-G-Oliver — APPROVED** [review]

File: src/components/pages/ParentContact/ParentContactContainer.js
Comment (MosharrafMusa) [reply src/components/pages/ParentContact/ParentContactContainer.js:25]: Appreciate! Thank you for your constructive feedback. It is for storing the info in the local storage. Because we have no specific info whether the data will goes to any database or use a third-party software to store the info.

**Pull request merged**
